scripts/webpack/webpack.config.dev.js

```diff
@@ -118,6 +118,70 @@ const webpackConfig = {
           { loader: 'ts-loader', options: { happyPackMode: true, transpileOnly: true } },
         ],
       },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              hmr: process.env.NODE_ENV !== 'production',
+            },
+          },
+          'css-loader',
+        ],
+      },
+      {
+        test: /\.(sass|scss)$/,
```

Comment:
repeat css loader injected in dev and prod code, consider spliting them into that can be reused between dev and prod config

Reply:
I think the reason why I separate prod and dev to avoid conflict for maintain in the future